controllers/kubeletconfig_controller.go

```diff
@@ -168,7 +168,8 @@ func mcoKubeletConfToKubeletConf(mcoKc *mcov1.KubeletConfig) (*kubeletconfigv1be
 func renderRTEConfig(namespace, name string, klConfig *kubeletconfigv1beta1.KubeletConfiguration) (*corev1.ConfigMap, error) {
 	conf := rteconfig.Config{
 		Resources: sysinfo.Config{
-			ReservedCPUs: klConfig.ReservedSystemCPUs,
+			ReservedCPUs:   klConfig.ReservedSystemCPUs,
+			ReservedMemory: findReservedMemoryFromKubelet(klConfig.ReservedMemory),
```

Comment:
What about ```klConfig.KubeReserved``` and ```klConfig.SystemReserved```?
We can have reserved memory there as well.

Reply:
true. We easily get the NUMA locality of the reserved memory from this value, and we skip all the computations, which we can safely assume have been performed either by cluster admin or by any other agent on their behalf. I'm not sure about adding extra logic concerning systemReserved and kubeReserved because in the cases we care about this value should be also accounted on a specific NUMA node and declared here; I'm reluctant to do accounting on the basis of the kubeReserved/systemReserved alone because this would mean duplicate some kubelet logic.

Reply:
ack